### ISSUE TITLE
Adjust geothermal_district_heating() for the intermediate years on scenario path

### ISSUE THREAD

**Issue by ulfmueller** [openego/powerd-data#323]
In p-e there is now also geothermal heat and electricity (from version 0.12 on). So we could use that (we would have to switch it on in the config file). The question is, how we did the georeferencing of these geothermal plants in egon for etrago. 

So, I think we have to 

- [ ] compare the approach in p-e with ours: https://github.com/openego/powerd-data/blob/9155828920eaf5cc4fabc1ecf3ce2418925d3a6d/src/egon/data/datasets/pypsaeur/__init__.py#L1451 VS. https://www.sciencedirect.com/science/article/pii/S0306261920312551
- [ ] decide which is better (from my first glance I would take the built-in approach from p-e)
- [ ] deal with the interpolation and georeferencing of the results afterwards (depending on how we decide, if we take the built-in p-e probably for the complete 2045 the interface has to be slightly adjusted due to different carrier naming i.e. "urban central geo thermal" (egon-data) vs. "geothermal heat" (p-e)

Overall, I would vote for the path which creates the less work (I would expect this fact to have a minor influence).

**Comment by ulfmueller** [comment]
I just saw that @ClaraBuettner  posted in #246 : 

> I checked how geothermal power plants are distributed within powerd-data. It considers if there is a geothermal potential next to the district heating grid. So I guess it makes sense to keep our implementation since the spatial distribution make a larger difference.

So, that would mean we would keep the option switched off in p-e. Our function basically just has to be run for each prenetwork on the path. But we do not have to interpolate because it is due to optimization and the path-consistency will be dealt with in p-e in the brownfield part. So @CarlosEpia you simply have to make sure that the function manipulates all prenetworks on the path in the execute function. Ok?

**Comment by CarlosEpia** [comment]
Sure. Done in 696ad17ee5e01dac7c0749aec98f453678f9f95f.

**Comment by ClaraBuettner** [comment]
Can we close this issue?

**Comment by CarlosEpia** [comment]
Yes. We made sure that all the function __geothermal_district_heating()__ manipulates all pre-networks.
